### PR TITLE
test(e2e): toast-notification

### PR DESCRIPTION
**Pull request by bosomt**

### CODE REVIEW

File: packages/integration-tests/projects/suite-web/tests/wallet/notification-toast.test.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        //
+        // Test execution
+        //
+        cy.log('turn on passphrase protection');
+        cy.getTestElement('@settings/device/passphrase-switch')
+            .click({ force: true })
+            .getConfirmActionOnDeviceModal();
+        cy.task('pressYes');
+        cy.getConfirmActionOnDeviceModal().should('not.exist');
+        cy.wait(1000);
```
Comment (mroz22) [inline packages/integration-tests/projects/suite-web/tests/wallet/notification-toast.test.ts]: wait not generally nice and also not needed here

**Review by mroz22 — CHANGES_REQUESTED** [review]

File: packages/integration-tests/projects/suite-web/tests/wallet/notification-toast.test.ts (hunk 1 above)
Comment (bosomt) [reply packages/integration-tests/projects/suite-web/tests/wallet/notification-toast.test.ts]: @mroz22 removed

**Comment by ondracja** [comment]
@bosomt Please rebase and fixup the commits into 1.

**Review by vdovhanych — CHANGES_REQUESTED** [review]
There is a merge commit in the pr. suggest closing this and creating a new pull request from a "fresh" branch with the desired changes.

**Comment by bosomt** [comment]
@vdovhanych sorry for mess, i was trying to merge two commits /removed one line from code/ ended with this, I will close this PR and create new one

**Pull request closed without merging**